magento/magento2
# Added missing save operation for bulk

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#23958
1. Fixes magento/magento2#31213

### Manual testing scenarios (*)

1. Create own bulk via - `$this->bulkManagement->scheduleBulk($bulkUuid, $operations, $bulkDescription, $userId);`
2. Check response via PostMan when consumer will do the job

### Questions or comments


### Contribution checklist (*)
